src/lib/display-mobile.ts

```diff
@@ -1,4 +1,6 @@
 export function isMobile() {
-  const displayMobile = window.innerWidth < 600
+  const displayMobile =
+    window.innerWidth < 450 ||
+    /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
```

Comment:
Use `import { detect } from 'detect-browser'`. See example in components/Chat/Chat.vue